services/src/main/java/org/keycloak/services/clientpolicy/context/BackchannelTokenRequestContext.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2021 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.services.clientpolicy.context;
+
+import javax.ws.rs.core.MultivaluedMap;
+
+import org.keycloak.protocol.oidc.grants.ciba.channel.CIBAAuthenticationRequest;
+import org.keycloak.services.clientpolicy.ClientPolicyContext;
+import org.keycloak.services.clientpolicy.ClientPolicyEvent;
+
+/**
+ * @author <a href="mailto:takashi.norimatsu.ws@hitachi.com">Takashi Norimatsu</a>
+ */
+public class BackchannelTokenRequestContext implements ClientPolicyContext {
```

Comment:
Is it possible to make sure that context objects related to CIBA are within the CIBA package structure?

Reply:
@mposolda I see, how about the following packages?
* `org.keycloak.protocol.oidc.grants.ciba.clientpolicy.executor` : CIBA's executors are located.
* `org.keycloak.protocol.oidc.grants.ciba.clientpolicy.condition` : CIBA's conditions are located.
* `org.keycloak.protocol.oidc.grants.ciba.clientpolicy.context` : CIBA's contexts are located.

Reply:
+1